projects/fsastorefrontlib/src/cms-components/sales-illustration/services/chart.service.ts

```diff
@@ -0,0 +1,76 @@
+import { Injectable } from '@angular/core';
```

Comment:
Services shouldn't be in the same place with components. We have`core/~some-feature~/facade/~some.service~` for this. I think we already have `chart-config` there. Maybe rename `chart-config` to just `chart` and move this service there?
Also, `@Injectable()` should have `providedIn` specified so it's tree-shakeable and can be lazily loaded if needs be. Also that means you don't have to specify this service inside `providers` property of any module (since you do it inside `@injectable()` decorator).